examples/safe-4337-passkeys/src/logic/safe.ts

```diff
@@ -77,7 +78,10 @@ function getWebAuthnSignerContract(provider: ethers.JsonRpcProvider, address: st
  * @returns The signer address.
  */
 function getSignerAddressFromPubkeyCoords(x: string, y: string): string {
-  const deploymentCode = ethers.solidityPacked(['bytes', 'uint256', 'uint256'], [WebAuthSignerBytecode, x, y])
```

Comment:
I _think_ there is a nicer way to do this with `new ethers.ContractFactory`:

```ts
const factory = new ethers.ContractFactory(WebAuthnSignerAbi, WebAuthnSignerBytecode);
const { data: deploymentCode } = factory.getDeployTransaction(x, y, WEBAUTHN_VERIFIER_ADDRESS);
```

It is fine as-is, if that doesn't work.

Reply:
Hmm, I'm not sure if it's nicer. While I agree it abstracts the lower-level details away, it feels a little bit too magic. Also, not everyone uses ethers, and it's much more likely that alternative applications have a method similar to `getCreate2Address` than `getDeployTransaction` that returns the `deploymentCode`, thus I believe it works better for an example app. What do you think?